modules/nw-nwop-viewing-config.adoc

```diff
@@ -0,0 +1,94 @@
+// Module filename: nw-nwop-viewing-config.adoc
+// Module included in the following assemblies:
+// * networking/configuring-network-operator.adoc
+
+[id='nw-nwop-viewing-config-{context}']
+= Viewing the Network Configuration in a Running Cluster
+
+
```

Comment:
You need an intro statement. Why do I want to do this? To confirm that the cluster's using the right network settings?